terraform/main.tf

```diff
@@ -23,12 +22,18 @@ limitations under the License.
 ///////////////////////////////////////////////////////////////////////////////////////
 // Create the primary cluster for this project.
 ///////////////////////////////////////////////////////////////////////////////////////
+data "google_container_engine_versions" "gke_versions" {
+  zone = "${var.zone}"
+}
 
 // Create the GKE Cluster
 resource "google_container_cluster" "primary" {
   name               = "stackdriver-monitoring-tutorial"
   zone               = "${var.zone}"
   initial_node_count = 2
+  monitoring_service = "monitoring.googleapis.com/kubernetes"
```

Comment:
Does this do the same as this https://cloud.google.com/kubernetes-engine/docs/how-to/monitoring

Reply:
No it enables the new one
https://cloud.google.com/monitoring/kubernetes-engine/